src-pwa/custom-service-worker.js

```diff
@@ -5,7 +5,13 @@
  */
 workbox.core.setCacheNameDetails({prefix: 'tptea'})
 self.workbox.precaching.precache(self.__precacheManifest)
-workbox.routing.registerRoute(new RegExp('/api'), workbox.strategies.staleWhileRevalidate())
+workbox.routing.registerRoute(({url}) => {
```

Comment:
find a way to just simply use RegExp, no need to search for words in url.search params
and pls help discard all changes in /dist